flink-test-utils-parent/flink-connector-testing-framework/src/main/java/org/apache/flink/connectors/test/common/utils/ConnectorJarUtils.java

```diff
@@ -0,0 +1,64 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.flink.connectors.test.common.utils;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import java.io.File;
+import java.io.FileNotFoundException;
+import java.util.Objects;
+
+/** Utilities for searching connector JARs. */
+public class ConnectorJarUtils {
+
+    private static File searchedJarFile = null;
+
+    private static final Logger LOG = LoggerFactory.getLogger(ConnectorJarUtils.class);
+
+    /**
+     * Search JAR file in target folder.
+     *
+     * @return JAR file
+     * @throws FileNotFoundException if JAR file is not found in target folder
+     */
+    public static File searchConnectorJar() throws FileNotFoundException {
```

Comment:
I think we have similar functionality already [here](https://github.com/apache/flink/blob/e32f0f82164512f632a533fad01ead6a12ac8152/flink-end-to-end-tests/flink-end-to-end-tests-common/src/main/java/org/apache/flink/tests/util/flink/FlinkResourceSetup.java#L75-L75). However, I'm not sure if the spec can be also applied to the `FlinkContainer`. @zentol ?